### PR TITLE
SR-IOV enhancement for RHOSP (SDN-1170)

### PR DESCRIPTION
**Pull request by mikemckiernan**
/hold

@atyronesmith , @jboxman , PTAL.  I doubt that I hit the correct high-level description that a RHOCP on RHOSP customer expects to find.  Please let me know the terms and wording that our customers will find compelling.

### CODE REVIEW

**Review by atyronesmith — APPROVED** [review]
lgtm

File: release_notes/ocp-4-7-release-notes.adoc [hunk 1]
```diff
@@ -171,6 +171,11 @@ For containers interacting with SR-IOV virtual functions (VFs) in Data Plane Dev
 
 The egress router CNI plug-in is introduced in Technical Preview. You can use the plug-in to deploy an egress router in redirect mode. This egress router provides parity for OVN-Kubernetes compared to OpenShift SDN, but for redirect mode only. The plug-in does not perform in HTTP proxy or DNS proxy modes, and this is a difference with the implementation for OpenShift SDN. For more information, see xref:../networking/ovn_kubernetes_network_provider/deploying-egress-router-ovn-redirection.adoc[Deploying an egress router pod in redirect mode].
 
+[id="ocp-4-7-sriov-networknodepolicy-netfilter"]
+==== SR-IOV network node policy enhancement for deployment with {rh-openstack-first}
+
+The SR-IOV Operator is enhanced to support an additional field, `spec.nicSelector.netFilter`, in the custom resource for an SR-IOV network node policy. You can use the new field to specify an {rh-openstack} network by the network ID. For more information, see xref:../networking/hardware_networks/configuring-sriov-device.adoc[Configuring an SR-IOV network device].
```
Comment (jboxman) [inline release_notes/ocp-4-7-release-notes.adoc:177]: probably s/SR-IOV Operator/SR-IOV Network Operator/

**Review by jboxman — APPROVED** [review]
1 comment, otherwise LGTM

File: release_notes/ocp-4-7-release-notes.adoc (hunk 1 above)
Comment (atyronesmith) [reply release_notes/ocp-4-7-release-notes.adoc:177]: SR-IOV Network Operator

**Pull request merged**